src/content/pages/accessibility.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: Accessibility at EuroPython
+subtitle:
+  Information about Accessibility at EuroPython as an attendee with maps of
+  venues, childcare and other accessibility information.
+---
+
+# Accessibility at EuroPython
+
+EuroPython strives to be welcoming, accessible and inclusive for all our
+guests attending the conference.
+If you have suggestions to improve accessibility, please let us know by contacting us at
+[helpdesk@europython.eu](mailto:helpdesk@europython.eu).
+
+
+## General
+
+For enquiries, visit our helpdesk at registration or look for volunteers wearing
```

Comment:
I'm thinking here maybe we could also provide an online way just in case someone needs help and they cannot reach out to volunteers (e.g. they are in a far away room) or their accessibility needs make it hard to go to the helpdesk.

Potentially they could reachout to servicedesk or maybe they could just ping somewhere in discord and then someone can reach out to them in private. Just ideas.

Reply:
Added a sentence regarding discord. Just not sure if we are fast enough here. And there is no direct ping.